client/src/features/projectsV2/fields/SlugFormField.tsx

```diff
@@ -17,44 +17,65 @@
  */
 
 import cx from "classnames";
-
-import { Controller } from "react-hook-form";
+import { ArrowCounterclockwise } from "react-bootstrap-icons";
 import type { FieldValues } from "react-hook-form";
+import { Controller } from "react-hook-form";
+import { Button, FormText, Input, InputGroup, Label } from "reactstrap";
 
-import { FormText, Input, Label } from "reactstrap";
-import type { GenericFormFieldProps } from "./formField.types";
+import type { GenericFormFieldPropsWithReset } from "./formField.types";
 
 export default function SlugFormField<T extends FieldValues>({
+  compact,
   control,
   entityName,
   errors,
+  isDirty,
+  resetFunction,
   name,
-}: GenericFormFieldProps<T>) {
+}: GenericFormFieldPropsWithReset<T>) {
+  const content = (
+    <Controller
+      control={control}
+      name={name}
+      render={({ field }) => {
```

Comment:
Get the value of `isDirty` here.

```suggestion
      render={({ field, fieldState }) => {  // then use fieldState.isDirty
```

Or
```suggestion
      render={({ field, fieldState: { isDirty } }) => {
```

Reply:
:+1:  https://github.com/SwissDataScienceCenter/renku-ui/pull/3399/commits/13f253c06b28b0ea5959aadcffed75e1fc0f2227